src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/RestoreOperation/RestoreDatabaseService.cs

```diff
@@ -0,0 +1,217 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.SqlServer.Management.Common;
+using Microsoft.SqlServer.Management.Smo;
+using Microsoft.SqlTools.ServiceLayer.Connection;
+using Microsoft.SqlTools.ServiceLayer.DisasterRecovery.Contracts;
+using Microsoft.SqlTools.ServiceLayer.TaskServices;
+
+namespace Microsoft.SqlTools.ServiceLayer.DisasterRecovery.RestoreOperation
+{
+    /// <summary>
+    /// Includes method to all restore operations
+    /// </summary>
+    public class RestoreDatabaseService
```

Comment:
>RestoreDatabaseService [](start = 17, length = 22)

maybe this should be named something like RestoreDatabaseHelper since Service has a specific meaning in this app (it's registered as a service on startup).  If this isn't acting like the other services in this regard let's please use a different suffix.